### PR TITLE
fixed color of status bar for pwa

### LINKED ISSUE #8283: [CAL-1498] dark <meta> theme on "standalone"
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name/theme-color

the white appbar should be dark on dark mode

steps to reproduce:

- on iOS, visit app.cal.com on safari 
- add to homescreen
- open 

image

<sub>[CAL-1498](https://linear.app/calcom/issue/CAL-1498/dark-meta-theme-on-standalone)</sub>

### PR DESCRIPTION
**Pull request by tap0212**
## What does this PR do?
- This PR fixes the issue raised by @PeerRich  in #8283 
- Follows the fix provided in the same issue

Fixes #8283 

## Before 

232170593-57e45b13-9f95-4dac-a74e-cca3281e74f3

## After
- ### iOS
- 

https://user-images.githubusercontent.com/40211415/235367080-37bbe479-497f-46bb-a82c-eca0bc13037f.mp4

- ### Android
image (1)
image

**Environment**: Staging(main branch) 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- on iOS, visit app.cal.com on safari
- add to homescreen
- open 
- try togging the theme

### CODE REVIEW

File: apps/web/pages/_document.tsx [hunk 1]
```diff
@@ -45,7 +45,8 @@ class MyDocument extends Document<Props> {
           <link rel="manifest" href="/site.webmanifest" />
           <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#000000" />
           <meta name="msapplication-TileColor" content="#ff0000" />
-          <meta name="theme-color" content="var(--cal-bg)" />
+          <meta name="theme-color" media="(prefers-color-scheme: light)" content="cyan" />
```
Comment (joeauyeung) [inline apps/web/pages/_document.tsx:48]: @sean-brydon for light mode is this the colour we want?

Comment (PeerRich) [reply apps/web/pages/_document.tsx:48]: definitely not cyan

Comment (PeerRich) [inline apps/web/pages/_document.tsx:48]:
```suggestion
          <meta name="theme-color" media="(prefers-color-scheme: light)" content="white" />
```

File: apps/web/pages/_document.tsx [hunk 2]
```diff
@@ -45,7 +45,8 @@ class MyDocument extends Document<Props> {
...
+          <meta name="theme-color" media="(prefers-color-scheme: dark)" content="black" />
```
Comment (PeerRich) [inline apps/web/pages/_document.tsx:49]:
```suggestion
          <meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1C1C1C" />
```

**Review by PeerRich — APPROVED** [review]

**Pull request merged**